config/Dockerfiles/JenkinsfileRelease

```diff
@@ -0,0 +1,182 @@
+/**
+ * CD LinchPin Release
+ *
+ * This job pulls down the git repository when a new version has been
+ * provided, tags the release version and pushes a new release to pypi.org
+ */
+
+import com.cloudbees.plugins.credentials.Credentials
+
+// Execution ID for this run of the pipeline
+executionID = UUID.randomUUID().toString()
+// Use repoName everywhere for dirs and repo values
+repoName = 'jenkins-test'
+
+
+env.ghprbGhRepository = env.ghprbGhRepository ?: 'herlo/' + repoName
```

Comment:
I know you are testing...  but make sure this changes for the final version of the patch.